bin/quota

```diff
@@ -479,23 +551,36 @@ function printQuota() {
 function reformatQuota() {
   local _quota_value="${1}"
   local _regex='^([0-9.,][0-9.,]*)([kMGTP]?)'
+  declare -A _unit2factors=(['k']='1' ['M']='2' ['G']='3' ['T']='4' ['P']='5')
+  declare -A _factors2unit=(['1']='k' ['2']='M' ['3']='G' ['4']='T' ['5']='P')
+  local _factor
   if [[ "${_quota_value}" =~ ${_regex} ]]; then
-    local _int="${BASH_REMATCH[1]}"
+    local _number="${BASH_REMATCH[1]}"
     local _unit="${BASH_REMATCH[2]}"
-    if [[ -z "${_unit:-}" && "${_int}" -gt 5 ]]; then
-      _int=$((${_int}/1000))
-      _unit='k'
-    fi
-    printf -v _formatted_number "%'.1f" "${_int}"
+    _number="${_number//,/}" # Remove all ',' thousand separators, which bc does not like.
+    local _int="${_number%%.*}"
```

Comment:
ervan uitgaande dat er maar 1 . (punt) in een number in staat is een %% misschien een overkill

Reply:
```
    local _int="${_number%.*}"
```
zou in theorie genoeg moeten zijn, maar de hamvraag is dan wat je wil dat gebeurt als er (onjuist) toch meer punten in zitten: een unhandled exception die de handel crasht of een mogelijk onjuiste waarde. Het was dat laatste, maar ik heb het nu aangepast...